index.js

```diff
@@ -414,3 +435,13 @@ process.on('SIGINT', () => {
 });
 
 module.exports = rcl;
+
+// The following statements are located here to work around a 
+// circular dependency issue occuring in rate.js
```

Comment:
Is it possible to break the circular dependency? It seems tricky to do so.

Reply:
The work-around for the circular dependencies is to revert back to the former implementation of rate.js. That version of rate.js used private functions or node and duplicated code from index.js to avoid importing index.js. 

I have no issue rolling back to that code to avoid the complicated import/require code placement as we have now. Say the word and I'll revert rate.js and no longer need the current workaround in index.js.

Reply:
I see, so let's keep this work around.